projects/curl.se/ca-certs/package.yml

```diff
@@ -0,0 +1,23 @@
+distributable: ~
+
+versions:
+  # maybe base it off the URL?
+  - 1.0.0
```

Comment:
the calver is `2022.07.19`

Reply:
yeah, I balked because the URL uses dashes, but `URL_VER=$(echo {{version.raw}} | tr -- . -)` gets it

Reply:
or you can do `{{version.major}}-{{version.minor}}` etc.

Reply:
oh I guess you can't because calver needs the leading zeros.

Time to add proper calver mustaches IMO!

Reply:
edit: I started looking into this and was then flummoxed by how to handle 4 components semvers. So I looked at Warp which has calver with times, but it starts with `v0` so I then became uninterested.

Reply:
K adding `{{version.raw[0]}}`, `{{version.raw[1]}}` etc.

Reply:
Like maybe it's not a great idea. But it's purely additive.